### PR TITLE
[20529] Fix CI documentation workflow label triggering

### PR DESCRIPTION
**Pull request by JesusPoderoso**
## Description

This PR fixes the label triggering of the documentation CI workflow. Before, it was set to avoid the workflow if the label `no-test` was added, but that was not the intended use. With this PR, `no-test` label triggers the job, while `skip-ci` label does not.

Furthermore, this PR also refactors the documentation-test workflow to use eProsima-CI. This PR depends on:

* eProsima/eProsima-CI#63

@Mergifyio backport 2.13.x 2.12.x 2.10.x 2.6.x

## Contributor Checklist
- [x] Commit messages follow the project guidelines. 
- **N/A** The code follows the style guidelines of this project. 
- **N/A** Tests that thoroughly check the new feature have been added/Regression tests checking the bug and its fix have been added; the added tests pass locally 
- **N/A** Any new/modified methods have been properly documented using Doxygen. 
- **N/A** Changes are ABI compatible. 
- **N/A** Changes are API compatible. 
- **N/A** New feature has been added to the `versions.md` file (if applicable).
- **N/A** New feature has been documented/Current behavior is correctly described in the documentation. 
    
- [x] Applicable backports have been included in the description.

## Reviewer Checklist
- [x] The PR has a milestone assigned.
- [x] The title and description correctly express the PR's purpose.
- [x] Check contributor checklist is correct.
- [x] Check CI results: changes do not issue any warning.
- [x] Check CI results: failing tests are unrelated with the changes.

### CODE REVIEW

**Review by JesusPoderoso — COMMENTED** [review]
[e69dbfe](https://github.com/eProsima/Fast-DDS/pull/4456/commits/e69dbfe5529166154b64dc67e096d947dac420e5) changes LGTM

File: .github/workflows/documentation-tests.yaml [hunk 1]
```diff
@@ -34,31 +34,46 @@ env:
...
+        uses: eProsima/eProsima-CI/multiplatform/install_python_packages@v0
+        with:
+          packages: vcstool
+          upgrade: false
+
+      - name: Install colcon
+        uses: eProsima/eProsima-CI/ubuntu/install_colcon@v0
+
+      - name: Download Fast DDS .repos
+        uses: eProsima/eProsima-CI/ubuntu/vcs_import@feature/vcs_import/skip_existing
```
Comment (JesusPoderoso) [inline .github/workflows/documentation-tests.yaml:72]: Please, remember to unset this before merging, once the patch release has been done in [eProsima-CI](https://github.com/eProsima/eProsima-CI)

```suggestion
        uses: eProsima/eProsima-CI/ubuntu/vcs_import@v0
```

**Review by JesusPoderoso — COMMENTED** [review]
Changes [7d50e8f](https://github.com/eProsima/Fast-DDS/pull/4456/commits/7d50e8f9dfde30399c5e69969798a363edee8421) and [8390649](https://github.com/eProsima/Fast-DDS/pull/4456/commits/83906493fa3159450bb689ad11ed3122e798bb47) LGTM

**Review by JesusPoderoso — COMMENTED** [review]
Commits from [77befa7](https://github.com/eProsima/Fast-DDS/pull/4456/commits/77befa72294562826a8c3f41bbcae485075b1814) to [693a6a0](https://github.com/eProsima/Fast-DDS/pull/4456/commits/693a6a0ef7fe354277bcee705ca16005eda49dab) LGTM 
( [77befa7](https://github.com/eProsima/Fast-DDS/pull/4456/commits/77befa72294562826a8c3f41bbcae485075b1814), [1d8501d](https://github.com/eProsima/Fast-DDS/pull/4456/commits/1d8501d81b5f2b2abc489b55a5ef29e1e426ffee), [4e11f46](https://github.com/eProsima/Fast-DDS/pull/4456/commits/4e11f46a03f1d18528406512a58ec799463ddadb), [693a6a0](https://github.com/eProsima/Fast-DDS/pull/4456/commits/693a6a0ef7fe354277bcee705ca16005eda49dab))

**Review by EduPonz — APPROVED** [review]

**Review by elianalf — APPROVED** [review]

**Comment by EduPonz** [comment]
@Mergifyio backport 2.13.x 2.12.x 2.10.x 2.6.x

**Pull request merged**